app/javascript/components/Greeting.js

```diff
@@ -0,0 +1,15 @@
+import { useSelector } from 'react-redux';
+
+function Greeting() {
+  const greetings = useSelector((state) => state.greeting);
+  return (
+    <div>
+      <h1>Some One Says:</h1>
+      <h2>
+        {greetings}
+      </h2>
+    </div>
```

Comment:
- [ ] Please 🙏🏽 check why this is not returning an HTML file but a JSON file instead for a better UI 👇🏽 

![image](https://github.com/demix007/hello-rails-react/assets/58642949/f3fc5f90-dac7-4d80-bc9e-6547c1950675)